.travis.yml

```diff
@@ -2,8 +2,6 @@ language: ruby
 bundler_args: ""
 sudo: false
 rvm:
-  - 2.0.0
-  - 2.1.8
```

Comment:
I need at least 2.1 for some projects ... can you ad an exclude instead and keep 2.1 ? ...